BlockSettleUILib/CreateTransactionDialog.cpp

```diff
@@ -191,9 +191,10 @@ void CreateTransactionDialog::closeEvent(QCloseEvent *e)
    e->ignore();
 }
 
-int CreateTransactionDialog::SelectWallet(const std::string& walletId)
+int CreateTransactionDialog::SelectWallet(const std::string& walletId, int type)
```

Comment:
Please use explicit type instead of int
UiUtils::WalletsType

Reply:
Sure, updated